plugins/sentinelone/help.md

```diff
@@ -1989,6 +2035,7 @@ To convert `threat` into an array use Type Converter Plugin
 
 # Version History
 
+* 8.1.0 - New actions: Fetch file for agent id. Run remote script
```

Comment:
```suggestion
* 8.1.0 - New actions: `Fetch file for agent ID` & `Run remote script`
```

Reply:
Updated to capitalise the ID but didn't add the backticks to keep consistent with previous messages.